builder/openstack/step_create_image.go

```diff
@@ -45,6 +45,7 @@ func (s *stepCreateImage) Run(ctx context.Context, state multistep.StateBag) mul
 	// Block Storage service volume or regular Compute service local volume.
 	ui.Say(fmt.Sprintf("Creating the image: %s", config.ImageName))
 	var imageId string
+	var blockStorageClient *gophercloud.ServiceClient
 	if s.UseBlockStorageVolume {
 		// We need the v3 block storage client.
 		blockStorageClient, err := config.blockStorageV3Client()
```

Comment:
watch out, out of this scope `blockStorageClient` will always be nil.
```suggestion
		blockStorageClient, err = config.blockStorageV3Client()
```

Reply:
Yes, missed it. Fixed :smile: